src/sbt-test/sbt-rat/apache-headers/src/main/scala/BSD3.scala

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2011-2013, Someone
```

Comment:
I'd prefer something more legal. How about `Copyright 1850 Abraham Lincoln` or something expired? ;)

Reply:
IANAL, but claiming something is copyright someone else doesn't feel right. Maybe we could switch to a file with a different license, maybe Creative Commons would have less issues? The test doesn't really care what the license is.

Reply:
Ah, well, perhaps a real copyright then? Or at least match the actual year. The file is trivial enough that I doubt copyright would actually apply to it besides the license itself.

Reply:
Will update to a real copyright. Agreed that copyright probably doesn't even really apply since this is so trivial.